examples/follow.aps

```diff
@@ -21,7 +21,7 @@ module FOLLOW[T :: var GRAMMAR[]] extends T begin
   pragma synthesized(items_predict);
   pragma inherited (items_follow);
 
-  circular attribute Grammar.grammar_follow : DeclarationTable;
```

Comment:
This seems wrong to me.  When we have a use of a variable we look it up in the table.  We may need to update a table entry for a symbol during fixpoint analysis.  Maybe there's a bug somewhere in circularity detection?